tests/yakus/afterkan.test.cpp

```diff
@@ -0,0 +1,69 @@
+#include <gtest/gtest.h>
```

Comment:
I'm in general worried about this test as im not sure the what the state will be. Did you look into the state functions to get an idea?

Reply:
I did. Integration testing would be required to verify those flags in the state are actually properly utilized

Reply:
Yes. However integration testing should be applied later to ensure those flags are being set properly